content/applications/marketing/marketing_automation/target_audience.rst

```diff
@@ -1,123 +1,123 @@
 ==================
-Target an audience
+Audience targeting
```

Comment:
I could be mistaken, but I believe we're trying to avoid any "ing" phrasing in the main doc h1 -- so, perhaps the original h1 would work best? Again, things may have changed since that information was shared with me. So, this title may work. worth checking into, either way. figured it was worth leaving a comment about. 👍

Reply:
This was [a suggestion](https://github.com/odoo/documentation/pull/7711#discussion_r1501237914) from ZST in my previous PR, I'll discuss and update as needed :)

Reply:
@ksc-odoo I spoke with ZST and you are correct that we're trying to avoid any "ing" phrasing in the h1, however, in this case "Audience targeting" is an industry term and is more recognizable than "Target an audience". 

So for the benefit of the _target audience_ 🤓 of this doc, I am going to leave the H1 as "Audience targeting" :)